### ISSUE TITLE
ANDROID Host url

### ISSUE THREAD

**Issue by ezraholm50** [Fmstrat/ownnote#238]
Hey great app thanks! I had some trouble with finding the right url for the android app.
I know i had something like https://domainname.com/remote.php/apps/ownnote but i cant seem to figure out what it is this time... hope someone is able to help.

Kind regards,

Ezra

**Comment by ezraholm50** [comment]
Anyone?

**Comment by improti** [comment]
I don't think you need to enter all of that, seems you only need the main part. On the other hand, I haven't got this app to work yet, so don't trust on me here ;) I used to get an error about JSON which now changed to telling me the URL wouldn't exist.

**Comment by ezraholm50** [comment]
Still cant get it to work. Anyone?

**Comment by ezraholm50** [comment]
I think i tried every combination possible, can anyone help me?

**Comment by DaveJ61** [comment]
I'm having this trouble too. I can't find any documentation to help either.

**Comment by improti** [comment]
You shouldn't need documentation actually. Does you URL work in the browser from the desktop? I had the wrong server-side app running (there are Notes and OwnNote) and just needed to switch over another client. Check exactly what software you are using.

**Comment by DaveJ61** [comment]
Yes, the owncloud url works fine in a browser (Firefox) on my mobile and on my desktop. The owncloud clients work fine too. I take it that the url is supposed to be the owncloud server url and not something extra added....?
I disagree with you regarding needing documentation. Every piece of software should have at least installation instructions no matter how simple.

**Comment by improti** [comment]
Note that you are posting on the server-side app OC/OwnNote here. I assume your problem is rather client-side. Maybe check if there is a better place to reporting client bugs? The app should tell ya I guess. The Android client doesn't seem to be in this repository, however, so this might simply be the wrong place. 

And I can't help because I switched OC/Notes with MyOwnNotes on Android (I wanted an OpenSource client available on F-Droid). But those are not compatible with OC/OwnNote and the according client that seems to be available on the Google Store only.

**Comment by ezraholm50** [comment]
The only question I have is what the remote url is to use ownnote on for
example the android app.

I tried alot of combinations like:

https://domain.com/remote.php/ownnote
https://domain.com

Also with various other words like index.php or apps added to the url but
nothing seems so work.

I'm sure one of the devs can tell me what to use. Thanks

Op wo 7 okt. 2015 18:39 schreef improti <notifications@github.com>:

> Note that you are posting on the server-side app OC/OwnNote here. I assume
> your problem is rather client-side. Maybe check if there is a better place
> to reporting client bugs? The app should tell ya I guess. The Android
> client doesn't seem to be in this repository, however, so this might simply
> be the wrong place.
>
> And I can't help because I switched OC/Notes with MyOwnNotes on Android (I
> wanted an OpenSource client available on F-Droid). But those are not
> compatible with OC/OwnNote and the according client that seems to be
> available on the Google Store only.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Fmstrat/ownnote/issues/238#issuecomment-146256921>.
>
-- 

Met vriendelijke groet,

E. Holm

**Comment by DaveJ61** [comment]
> Note that you are posting on the server-side app OC/OwnNote here. I assume your problem is rather client-side. Maybe check if there is a better place to reporting client bugs? The app should tell ya I guess. The Android client doesn't seem to be in this repository, however, so this might simply be the wrong place.

I hadn't thought of this but I have checked and the support links all come back here.
Anyway, as with Ezra, all I need is the url required to access the server. From the android client, it cannot even connect or see the server.
From looking at this repository, it appears that development may have stopped several months ago.
I also noted that issues have been posted with nobody replying to them. I might just pass on this and use notes instead.

**Comment by Fmstrat** [comment]
It should just be like in the example provided if the URL field is blank. It's whatever the root directory of ownCloud is. So if you use http://mydomain.com/owncloud, but that into ownNote. It looks like your trying to use the term "ownnote" instead of "owncloud" in the URL.

**Comment by ezraholm50** [comment]
My owncloud is accessoires via https://domein.com thats it. If i insert this in ownnote app it gives the above errors. Qlueless...

**Comment by ezraholm50** [comment]
Sorry android wont let me edit above post. Accessoires must be accessed and domein domain ;)

**Comment by jpellegrini** [comment]
Hello,
I am using nextcloud on a server, and ownNotes won't sync with it. I have used the base URL only, as described by you. The app successfully authenticates, shows the animated sync icon, but no notes are actually transferred to or from the server.

**Comment by enoch85** [comment]
@jpellegrini Did it work with ownCloud before the switch to Nextcloud?

@Fmstrat Where is the support for this app, gone?
